Source/core/Time.h

```diff
@@ -26,7 +26,44 @@
 namespace WPEFramework {
 namespace Core {
     class EXTERNAL Time {
-    private:
+
+    public:
+        class TimeAsLocal {
+        private:
+            friend class Time;
+
+            TimeAsLocal(Time time)
```

Comment:
Discussed with Pierre, because of his use case he needs an default constructor as well so having this one private with a friend does not add much value anymore. So I propose to make this one public and also add the default constructor
Also please pass it as a const& or if you want to make it even better you could also add a move constructor :)

Reply:
Added default constructor and moved TimeAsLocal(Time time) as per comment